staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -77,8 +77,6 @@ type Event struct {
 	// AuditLevel at which event was generated
 	Level Level
 
-	// Time the request reached the apiserver.
-	Timestamp metav1.Time
```

Comment:
> I prefer use CreationTimestamp as "request timestamp", this time is also when the audit event is initialized.

I totally understand that both are valid points, but I'd like to argue for a little bit to be confident about either solution we choose :)

If you look at the audit events stream from the outside, it's just a stream of objects. Each objects is associated with another "umbrella" entity (request), but it's more than that, it's a separate entity. Object's metadata in Kubernetes tend to describe something about the object itself, not about the "umbrella" entity for this objects, for example pod's metadata describes the pod itself, not the deployment this pod was created for. So, it seems logical to me to have the information about the audit event itself in the object's metadata, not about the process this event was created for